santosfamilyfoundation/RuinaMain
# [RUIN-58] Enable saving to local device with multiple file formats at once

# Description
Multiple exports work for all pdf, html, and excel files. Although there is a problem with their encoding as they don't seem to save properly (i.e. it is saved in the file system, but you can't open the files because their encoding is wrong), I can continue to fix that another time. 

# Testing
1. Checkout the branch
2. Go to export and click it
3. Try checking any combination of files and see if it shows the file saved successfully message.
4. Check if the files actually exist in the file system.
